Gemfile

```diff
@@ -2,7 +2,14 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'mysql2', '~> 0.5.3'
-gem 'pg', ">= 0.18", "< 2.0"
 gem 'rubocop', '~> 1.5'
+
+gem 'mysql2', '~> 0.5.3', platform: :mri
+gem 'pg', ">= 0.18", "< 2.0", platform: :mri
+gem 'memcached', '~> 1.8.0', platform: :mri
+gem 'memcached_store', '~> 1.0.0', platform: :mri
+gem 'dalli', '~> 2.7.11'
+gem 'cityhash', '~> 0.6.0', platform: :mri
+
 gem 'byebug', platform: :mri
+gem 'stackprof', platform: :mri
```

Comment:
Why are we specifying `platform: :mri` in these gems?

Reply:
Yeah, these are native extensions, so we can just not install them when not using MRI.